src/resolvers/jazzicon.ts

```diff
@@ -0,0 +1,19 @@
+import { JSDOM } from 'jsdom';
+import sharp from 'sharp';
+import jazzicon from '@metamask/jazzicon';
+import { resize } from '../utils';
+import { max } from '../constants.json';
+
+const dom = new JSDOM('');
+global.document = dom.window.document;
+
+export default async function resolve(address) {
+  const { innerHTML, style } = jazzicon(64, parseInt(address.slice(2, 10), 16));
```

Comment:
Just changing this otherwise it make low quality on large size image
```suggestion
  const { innerHTML, style } = jazzicon(500, parseInt(address.slice(2, 10), 16));
```